### PR TITLE
Gem docs updated

### PR DESCRIPTION
**Pull request by Sammers21**
Gem docs have been updated.

Related to #361

### CODE REVIEW

File: examples/gem/README.md [hunk 1]
```diff
@@ -2,24 +2,40 @@
...
+
+Before uploading your gems, you should obtain a key for authorization. A base64 encoded
+login:password would be a valid key:
+
+```bash
+export GEM_HOST_API_KEY=$(echo -n "hello:world" | base64)
 ```
 
-Publish a gem:
+In order to upload a `.gem` file into, use `gem push` command:
```
Comment (olenagerasimova) [inline examples/gem/README.md:27]: @Sammers21 seems like several words before comma are missing

File: examples/gem/README.md [hunk 2]
```diff
@@ -2,24 +2,40 @@
...
 
 ```bash
 $ gem push my_first_gem-0.0.0.gem --host http://localhost:8080/gem
 ```
 
-Install a gem:
+In order to install an existing gem , use `gem install` command:
```
Comment (olenagerasimova) [inline examples/gem/README.md:33]: @Sammers21 please remove space before comma

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@Sammers21 two comments, take a look please

**Review by olenagerasimova — APPROVED** [review]
@Sammers21 thanks
@olegmoz looks good to merge

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#590` is not assigned, can't get performer

**Pull request merged**